### PR TITLE
fix typo and add missing information to the docs

### PR DESCRIPTION
**Pull request by andersy005**

### CODE REVIEW

**Comment by cisaacstern** [comment]
LGTM, @andersy005, thanks! Feel free to merge whenever you're happy with this. (Idk if you're planning to add more or not, so I'll hold off on merging right now.)

**Comment by cisaacstern** [comment]
(I deleted the review app for this PR on the Heroku dashboard, since this is a docs PR and we don't need a review app.)

File: docs/README.md [hunk 1]
```diff
@@ -202,6 +202,12 @@ $ python3 scripts.develop/generate_api_key.py ${Your GitHub Username} local
 
 Now, `secrets/config.pforge-local-${Your GitHub Username}.yaml` should contain an api key.
 
+In addition, you will need to create a configuration file with the bakery information by copying the `bakeries/pangeo-ldeo-nsf-earthcube.pforge-local-cisasacstern.yaml` file to `bakeries/pangeo-ldeo-nsf-earthcube.local-${Your GitHub Username}.yaml`:
+
+```console
+$ cp bakeries/pangeo-ldeo-nsf-earthcube.pforge-local-cisasacstern.yaml bakeries/pangeo-ldeo-nsf-earthcube.local-${Your GitHub Username}.yaml
+```
```
Comment (andersy005) [inline docs/README.md:209]: @cisaacstern, let me know if this isn't in the right place....

Comment (cisaacstern) [reply docs/README.md:209]: @andersy005, this should probably under it's own subheading, right before **Start the Server** below.

Something like:

```
## Bakeries config

The last step before starting the app server is to setup bakeries config for your local app instance.

...
```

Comment (andersy005) [reply docs/README.md]: Great! 👍

File: docs/README.md [hunk 2]
```diff
@@ -295,6 +289,14 @@ The `"url"` field in the response should refect the `PROXY_URL` you passed to th
...
 time by re-running this last script with a different argument.
 
+## Bakeries config
+
+The last step before starting the app server is to setup bakeries config for your local app instance. You can create a configuration file with the bakery information by copying the `bakeries/pangeo-ldeo-nsf-earthcube.pforge-local-cisasacstern.yaml` file to `bakeries/pangeo-ldeo-nsf-earthcube.local-${Your GitHub Username}.yaml`:
+
+```console
+$ cp bakeries/pangeo-ldeo-nsf-earthcube.pforge-local-cisasacstern.yaml bakeries/pangeo-ldeo-nsf-earthcube.local-${Your GitHub Username}.yaml
+```
+
```
Comment (cisaacstern) [inline docs/README.md:299]:
````suggestion
Finally, decrypt the secret args for this bakery:

```console
$ sops -d -I secrets/bakery-args.pangeo-ldeo-nsf-earthcube.yaml
```
> **Note from @cisaacstern**: This entire bakeries config setup (which I wrote) is _far to convoluted_. These 
instructions are designed to get your local app running under the current paradigm. This should be revisited and 
simplified considerably. For further discussion on this, see https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/129.
````

Comment (cisaacstern) [reply docs/README.md:299]: @andersy005, one last suggestion here

**Comment by cisaacstern** [comment]
Thanks, @andersy005!

**Pull request merged**